sympy/simplify/radsimp.py

```diff
@@ -966,6 +966,8 @@ def handle(expr):
             n *= num
             d *= num
             d = powdenest(_mexpand(d), force=symbolic)
+            if d.has(S.Zero, nan, zoo):
```

Comment:
Does the problem already exist with `num` a few lines earlier?

Reply:
@smichr No , the problem exists only after line 968.